card/mod/01_core/set/all/tracked_attributes.rb

```diff
@@ -8,7 +8,9 @@ def assign_attributes args={}, options={}
     @subcards = extract_subcard_args! args
     reset_patterns
   end
-  super args, options
+  params = ActionController::Parameters.new(args)
+  params.permit!
+  super params
```

Comment:
we should probably put some comments here; I suspect some rails folks might challenge us on this decision if they dig deeply enough.

I suspect it's the right way to go for now...